model_analyzer/config/input/config_command.py

```diff
@@ -109,6 +111,7 @@ def set_config_values(self, args):
                 raise TritonModelAnalyzerException(
                     f'Config for {value.name()} is not specified. You need to specify it using the YAML config file or using the {flags} flags in CLI.'
                 )
+        self._update_perf_latency_to_p99(self._fields)
```

Comment:
I think it is a very custom logic and should not be placed here. Why do you need all this custom logic for `perf_latency` to `perf_latency_p99` alias?

Reply:
Yes, this logic is problematic. It's goal is to replace `perf_latency` with `perf_latency_p99` when parsing the input configs. 
I'll either refactor it into `config_command_analyze.py` and `config_command_report.py` or just have another metric named `perf_latency`, depending on which works better.

Reply:
Can we use autofill function and not remove the `perf_latency` from the list of configs? If the user has provided `perf_latency` we will set the same value for `perf_latency_p99` and if the user has provided a value for `perf_latency_p99` we set it for `perf_latency`.

Reply:
This is updated. Config will automatically convert all `perf_latency` into `perf_latency_p99`. A warning will be printed once if `perf_latency` is used anywhere.

Locations updated are:
  - `config_command_report.py`: `plots`, `plots` of each `report_model_configs`
  - `config_command_analyze.py`: `inference_output_fields`, `objectives`, `constraints`, `objectives` and `constraints` of each `analysis_models`
  - `config_command_profile.py`: `objectives`, `constraints`, `objectives` and `constraints` of each `profile_models`

Reply:
3 unit tests are added to the bottom of `test_config.py` for each case.